src/services/GraphApplication.ts

```diff
@@ -0,0 +1,61 @@
+import { GraphManager } from './GraphManager';
+import {
+  getStrongestModel,
+  getFasterModel,
+  groqChatMixtral,
+  groqChatSmallLlama,
+  anthropicSonnet,
+  anthropicOpus,
+  createAgent,
+  anthropicHaiku,
+  groqChatLlama,
+  createPlanner,
+} from '../models/Models';
+import {
+  calculatorTool,
+  emailTool,
+  rewardTool,
+  filterTool,
+  eventTool,
+  tableTool,
+  chartTool,
+  infoCardTool,
+  pageHtmlTool,
+  sqlQuery,
+  segmentTool,
+  organizeItemTool,
+} from '../models/Tools';
+import Logger from '../utils/Logger';
+
+export class GraphApplication {
+  private graphManager: GraphManager;
+  private outputHandler: (type: string, message: string, ws: WebSocket) => void;
+
```

Comment:
If we have the output handler as a variable inside graph application, agentFunction should also be one, can we make that change in this PR

Reply:
I think this should be another PR to make it cleaner